software/artalk.yml

```diff
@@ -0,0 +1,13 @@
+name: "Artalk"
+website_url: "https://artalk.js.org/"
+source_code_url: "https://github.com/ArtalkJS/Artalk"
+description: "A self-hosted comment system built in Golang, providing a lightweight and highly customizable solution for adding comments to your website."
```

Comment:
```suggestion
description: "A self-hosted comment system built in Golang, providing a lightweight and highly customizable solution for adding comments to your website. (documentation in Chinese)."
```

Reply:
The English docs are actually available too. It might be a cache issue. Could you check it out again? Thanks!